.changeset/ten-candles-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+ "@primer/react": patch
```

Comment:
We usually release the docs changes without a changeset but did you want to announce this in the release notes?

Reply:
That was my plan. I was getting a warning though because there wasn't a changeset?

Reply:
Yeah the CI job will fail if there is no changeset. You could either add a changeset like you did here or you can add `skip changeset` label.